app/models/external_url.rb

```diff
@@ -0,0 +1,4 @@
+class ExternalUrl < ApplicationRecord
+  belongs_to :refers_to, :polymorphic => true
+  belongs_to :user
+end
```

Comment:
Add basic validation that the `url` property is actually a valid url.

Example: https://github.com/ManageIQ/manageiq/blob/master/app/models/git_repository.rb#L6

Reply:
Added, but w/o the schema limitation.